### ISSUE TITLE
Ambiguous ChEBI mappings

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#138]
Some ChEBI entries map to multiple PubChem and ChEBML entries. When loading these resource files into dictionaries, one entry is picked up and the others are ignored. Someone should look at these and see why these ambiguities exits and if there is a good way to prioritize which mapping to choose.

**Comment by johnbachman** [comment]
Can you give some examples? I'm wondering if this is only for the ones that are for classes of things, like "Kinase inhibitors".

**Comment by bgyori** [comment]
Below is a list of ambiguous entries. The first column is the ChEBI ID, the second and third are PUBCHEM IDs (note that in the current state of things, compound IDs and substance IDs can both appear here and are not differentiated in any way).
```
3504 87305645 96099266
3856 160711983 172120884
4946 160710300 171571832
5776 160711189 171571833
6698 136349241 172120885
10038 160711702 172120886
10353 160711511 223438276
10429 160709810 162012068
11515 8146716 93581525
15467 121269965 8143338
15469 85240320 99313653
15577 58106738 8145802
15701 123058728 160645907
16055 160709399 8145029
16126 135610377 24398008
16271 136349393 8143455
16870 8145747 92741042
17013 160962786 8145733
17038 57581474 8144710
17135 162169346 171571834
17197 11533126 162169359
17277 81058753 81058761
17392 123058730 160645906
17439 8144844 99437568
17631 111978144 8144072
17785 8145325 92741778
17810 85164689 92741314
17950 135610192 99313678
18347 135610263 8144529
18348 56464096 8145455
20833 223445492 252149670
22788 92729978 99313687
28066 104222399 8144404
28103 163835680 164174508
28107 10318900 121269854
28323 163835630 223438372
28353 163835685 223438374
28454 121269913 252149681
28610 164174758 223438381
28733 135668154 164174673
29503 163725485 163835620
30041 160709401 8145270
30042 160709400 8145269
31111 160711814 163626652
31998 26744233 56394948
32359 172121088 8147834
33196 14717811 49658847
33855 253662781 255509810
34778 160712527 223438442
36528 124403845 50139201
36702 124403633 92741551
36707 135610739 99437367
45996 223438479 8147972
46229 223438480 57581424
50043 135610928 49742745
50998 160644804 56394811
51013 56394810 57304837
52402 58106724 85164744
53028 85240324 99437564
53604 252162763 87246475
55376 87351936 99437544
57258 92741115 93581526
57649 125258855 99319310
57970 163726004 92741296
58086 103158386 99437451
58178 104222482 92741310
58456 92741373 92741501
58856 160644890 92741666
58909 99319423 99437575
59226 242084124 92741711
59338 103158248 92741919
60047 123058977 99319477
60059 160709402 99319483
60243 123058920 99364999
60522 223448152 99437570
60634 103158220 104222387
60998 104222543 124349142
61695 121270033 223442613
62071 123058744 49742813
62097 123058809 160962867
62190 123058866 87351980
62490 124403719 92741312
62744 160962889 162012087
62990 160645859 160962783
63528 135610817 26744298
64722 136349195 252162851
64771 136349157 163835723
64771 163835723 92741502
64872 136349279 171572122
65131 160644883 163725736
65273 160644675 8148008
65304 160644676 92741313
65646 160709932 160710570
65954 160709740 160710397
66292 160709713 160710958
66293 160709692 160710872
66994 160645637 160963266
67090 160645351 223444589
67666 160711118 160713300
68546 160645759 163726036
68743 160710591 223438502
68744 160710318 223438503
68774 160712019 164347273
69403 135668343 160712117
69404 135668344 160712407
69431 160711288 160712135
69994 160712622 160712953
71502 160709390 92741667
72829 162012370 163835633
72999 162169220 164174668
73002 162169228 164174628
73003 162169250 99437590
73008 162169204 163726027
73269 162169351 163425729
73315 162169379 92741305
73858 163725961 164174515
73999 163626702 163726001
74000 163626850 163725966
74546 163725696 223447609
74917 163725972 163835709
75063 163835430 163835643
75067 163835576 163835713
75137 163835637 163835689
75161 163835644 163835655
75173 163835646 163835682
75185 160645193 164174517
75243 163725784 163835705
75825 164347430 223444019
76225 170474399 223736556
77288 162012250 223442796
77292 223442799 252150482
77506 136349049 223438511
77919 163835722 223443334
78084 223443462 85735867
78462 223443799 223448579
78784 223444050 223448578
82639 223447370 223447624
83036 223445995 223447708
83569 223440605 223736270
83571 223446976 223736271
83792 223448516 252150513
83957 223443716 223736707
84199 223446972 223736272
84822 223443158 242083972
84945 223445353 242083813
85101 249807297 92741303
85238 249807254 249807256
85260 223441074 249807018
85533 223438581 252149688
85592 223448597 252149689
85593 252149944 85312168
86025 252150313 8147790
86381 223438662 252162377
86389 223438661 252162378
86409 223446474 252162379
86476 223442376 252162380
87044 223445296 252162858
87065 223448119 252162756
87399 223445256 252237802
87426 223441720 252237803
87665 223447247 252237804
87755 160711353 253662622
88014 223448046 253662624
88102 223446232 255509811
90222 223445331 255509815
90438 242084104 312698196
91030 223440233 312530903
131527 223441884 85735284
131759 223446374 85735508
132223 223445531 85332042
132853 223441107 85332664
244418 85437576 85656730
435764 160712567 85626378
546807 135610477 87250284
``

**Comment by bgyori** [comment]
Fixed in #865
